src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3617,6 +3618,34 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   </Target>
 
+  <!--
+    ============================================================
+                                        _GenerateSuggestedBindingRedirectsCache
+    Generate a file used to track suggested binding redirects between builds.
+    @(SuggestedBindingRedirects) never contains a file on disk, so create a file
+    that contains a hash of the items to prevent `GenerateBindingRedirects`
+    from running every build.
+
+    See https://github.com/dotnet/msbuild/issues/5943 for details.
+    ============================================================
+    -->
+  <Target Name="_GenerateSuggestedBindingRedirectsCache" Condition="'$(DesignTimeBuild)' != 'true' and '$(BuildingProject)' == 'true'" DependsOnTargets="ResolveAssemblyReferences">
```

Comment:
This feels like a normal bugfix. Do we need a changewave? I think not but want to put it out there.

Reply:
It doesn't warrant a changewave imo. This doesn't break anyone and in the worst case a target is running all of the time, which is the same behavior.